src/components/Content/Content.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import { ReactElement, useState } from "react";
+import { ContentProps, ContentType } from "./ContentTypes";
+
+// Components
+import Button from "../Button";
+import Chip from "../Chip";
+import {
+	MainWrapper,
+	Title,
+	TitleWrapper,
+	ButtonsWrapper,
+	Label,
+	Value,
+	LabelValueWrapper,
+	Paragraph,
+	ChipsWrapper,
+	LabelWrapper,
+	ValueFile,
+	ContentRow,
+	ContentColumn,
+	ParagraphWrapper,
+} from "./Content.styled";
+import EditIcon from "@mui/icons-material/Edit";
+import AddIcon from "@mui/icons-material/Add";
+
+const Content = (props: ContentProps): ReactElement => {
+	const { title, onClickEdit, content, onClickAdd } = props;
+	const [showOrHide, setShowOrHide] = useState(content.length < 3);
+
+	/**
+	 * Toggles the state use to show or hide the content.
+	 */
+	const showOrHideDetails = () => {
+		setShowOrHide(!showOrHide);
+	};
+
+	const contentForDisplay = showOrHide ? content : content.slice(0, 3);
+
+	/**
+	 * Renders the content of type "labelValue"
+	 * @param contentItem 
+	 * @returns The content with a sctructure of label: value
+	 */
+	const labelValueContent = (contentItem: ContentType) => (
+			<LabelValueWrapper>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 tabs but found 3.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9571838)
```suggestion
		<LabelValueWrapper>
```